sdk/Lusid.Sdk.Tests/tutorials/Instruments/ExoticExamples.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Lusid.Sdk.Model;
+using Lusid.Sdk.Tests.Utilities;
+using Lusid.Sdk.Utilities;
+using LusidFeatures;
+using NUnit.Framework;
+
+namespace Lusid.Sdk.Tests.Tutorials.Instruments
+{
+    [TestFixture]
+    public class ExoticExamples: DemoInstrumentBase
+    {
+        internal override void CreateAndUpsertMarketDataToLusid(string scope, ModelSelection.ModelEnum model, LusidInstrument instrument)
+        {
+        }
+
+        internal override LusidInstrument CreateExampleInstrument() => InstrumentExamples.CreateExampleExotic();
+
+        internal override void GetAndValidatePortfolioCashFlows(LusidInstrument instrument, string scope, string portfolioCode, string recipeCode, string instrumentID)
+        {
+            var cashflows = _transactionPortfoliosApi.GetPortfolioCashFlows(
+                scope: scope,
+                code: portfolioCode,
+                effectiveAt: TestDataUtilities.EffectiveAt,
+                windowStart: new DateTimeOrCutLabel(new DateTimeOffset(2000, 01, 01, 01, 0, 0, 0, TimeSpan.Zero)),
+                windowEnd: new DateTimeOrCutLabel(new DateTimeOffset(2050, 01, 01, 01, 0, 0, 0, TimeSpan.Zero)),
+                asAt:null,
+                filter:null,
+                recipeIdScope: scope,
+                recipeIdCode: recipeCode).Values;
+            
+            Assert.That(cashflows.Count, Is.EqualTo(2));
+            
+            _instrumentsApi.DeleteInstrument("ClientInternal", instrumentID);
+            _portfoliosApi.DeletePortfolio(scope, portfolioCode);
+        }
+
+        [LusidFeature("F22-10")]
+        [Test]
+        public void ExoticCreationAndUpsertionExample()
```

Comment:
this needs some commentary as to what an exotic is, or link to an appropriate KB article

Reply:
We probably want an appropriate link/documentation/KB article etc for all instruments (not just exotics). I'm not sure where to find them, do you know where?